misc.Rmd

```diff
@@ -2,56 +2,71 @@
 
 ```{r, echo = FALSE}
 source("common.R")
-status("restructuring")
+status("polishing")
 ```
 
 ## Introduction
 
-There are five other directories that are valid top-level directories.
-They are rarely used:
+The first two chapters in this part cover the two most obvious things that people distribute via an R package: functions (@sec-r) and data (\@sec-data).
+But that's not all it takes to make an R package.
+There are other package components that are either required, such as a `DESCRIPTION` file, or highly recommended, such as tests and documentation.
+
+The next few parts of the book are organized around important concepts: dependencies, testing, and documentation.
+But before we dig in to those topics, this chapter demystifies some package parts that are not needed in every package, but that are nice to be aware of.
+
+## Other directories
+
+Here are some top-level directories you might encounter in an R source package, in rough order of importance and frequency of use:
+
+-   `src/`: source and header files for compiled code, most often C and C++.
+    This is an important technique that is used to make R packages more performant and to unlock the power of external libraries for R users.
+    As of the second edition, the book no longer covers this topic, since a truly useful treatment of compiled code requires more space than we can give it here.
+    The tidyverse generally uses the [cpp11 package](https://cpp11.r-lib.org) to connect C++ to R and [Rcpp](https://www.rcpp.org) is the most well-established package for integrating R and C++.
 
 -   `inst/`: for arbitrary additional files that you want include in your package.
     This includes a few special files, like the `CITATION`, described below.
 
+-   `tools/`: auxiliary files needed during configuration, usually found in the company of a `configure` script.
+    We discuss this more below.
```

Comment:
This is the last bullet point where we promise more info below. Everything below here in the list is NOT elaborated upon. The bullet point is all these topics get.

Reply:
Use section ref?

Reply:
done